src/app/server/Server.cpp

```diff
@@ -314,15 +319,41 @@ static CHIP_ERROR OpenPairingWindowUsingVerifier(uint16_t discriminator, PASEVer
     params.SetPASEVerifier(verifier);
 #endif // CONFIG_NETWORK_LAYER_BLE
 
+    if (resetAdmins == ResetAdmins::kYes)
```

Comment:
Why do we want to reset the current admin table while we are opening the pairing window for PASE.